charts/workflow-controller/values.yaml

```diff
@@ -1,17 +1,14 @@
 # Default values for workflow-controller.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-replicaCount: 1
-namespace: default
 serviceAccount: workflow-controller
 logLevel: 2
-api:
-  groupName: dag.example.com
+apiGroupName: dag.example.com
 image:
   registry: ""
   account: "sdminonne"
   tag: "latest"
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
by doing this change, the e2e test will not work: currently, we build the image on the same docker daemon that is used to run the Kubernetes cluster. And so, with `IfNotPresent` the daemon will not try to download a new version from dockerhub.

It makes sense to replace: 'IfNotPresent' by 'Always' but in this case, you should overwrite the value when we run the `helm install` command.

Also in a next future PR, that will add the support of Github release. The Helm chart will be generated and published with a specific version instead of `latest`

Reply:
Agree. Will fix.